hydromt/raster.py

```diff
@@ -2340,28 +2353,61 @@ def to_slippy_tiles(
         }.get(driver.lower(), {})
         kwargs = {**kwargs0, **kwargs}
 
+        # Setting up information for zoomlevel calculation and
+        # determination of tile windows
+        # This section is purely for the resolution
+        bounds_4326_clip = list(obj.raster.transform_bounds("EPSG:4326"))
+        bounds_4326_clip[:2] = map(
+            max,
+            zip(bounds_4326_clip[:2], (-180, -y_ext)),
+        )
+        bounds_4326_clip[2:] = map(
+            min,
+            zip(bounds_4326_clip[2:], (180, y_ext)),
+        )
+        obj_clipped_to_pseudo = obj.raster.clip_bbox(
+            bounds_4326_clip,
+            crs="EPSG:4326",
+        )
```

Comment:
Wouldn't this do the same? Or am I missing something? 
```suggestion
        obj_clipped_to_pseudo = obj.raster.clip_bbox(
            [-180, -y_ext, 180, y_ext],
            crs="EPSG:4326",
        )
```

Reply:
No, I just wrote it way too explicitly..., kind of laughed when I saw this.